src/frontend/pages/game.tsx

```diff
@@ -0,0 +1,148 @@
+import Head from 'next/head';
+// @ts-ignore
+import styles from '../styles/Game.module.css';
+import Slider from 'react-input-slider';
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+export default function Home() {
+  const [takeVal, setTakeVal] = useState(10);
+
+  const inputOnChange = (eventVal: string) => {
+    const intVal = parseInt(eventVal);
+
+    // TODO: You can't type a '-', not sure we need this first condition
+    if (intVal < 0) {
+      alert('Input cannot be negative');
+      setTakeVal(0);
+    } else if (intVal > 10) {
+      alert('Input cannot be greater than 10');
+      setTakeVal(takeVal);
+    } else {
+      setTakeVal(intVal);
+    }
```

Comment:
The min/max only handles if the user is using the up/down buttons, not typed input :(

Reply:
If that's the case then is there a reason we need input both from the slider and a text box? Like would it be better just to have the slider, so that the user physically can't enter values we don't want?

Reply:
hmm, we didn't consider that, we just thought they might prefer being able to manually input it as well. Let's discuss this today at the meeting.

Reply:
@aislinblack i'm a huge fan of having both options there, as long as they're always in sync with each other. Sometimes sliders can be a pain to interact with when you just want an exact value. Maybe for just 10 this isn't as applicable, but when it comes to bigger numbers it's definitely important. We can extrapolate this discussion to Zoom/on my action bar branch